### PR TITLE
Fixed: reduce margins and only pay attention to menu anchors

### PR DESCRIPTION
**Pull request by james2doyle**
Tried to fix the issues with the headers. Give this a shot now. The changes look at the nav to figure out how to track the titles. I also tweaked the ratio of the viewport. Might need some more tweaks but the logic is slightly better now

### CODE REVIEW

File: resources/js/theme-default.js [hunk 1]
```diff
@@ -14,16 +14,21 @@ document.addEventListener('DOMContentLoaded', function() {
 
         throttledUpdateHash(entries[0].target.id);
     }, {
-        rootMargin: '-8% 0px -8% 0px', // shrink the intersection viewport
+        rootMargin: '0px 0px -12% 0px', // shrink the intersection viewport
```
Comment (james2doyle) [inline resources/js/theme-default.js:17]: I think we might need to use `rem` here in order to make all screens, no matter the size, work nicely. I will take a crack at that

**Comment by shalvah** [comment]
Welp, deleted the branch accidentally. Please reopen to master.

**Comment by shalvah** [comment]
Or never mind, pushed it back.

**Comment by shalvah** [comment]
@james2doyle is there a reason we need the IntersectionObserver? Removed it and the browser handles the hash changes just fine. Removing it for now.

**Comment by james2doyle** [comment]
@shalvah It was being used to track the scrolling so that the headers were triggering the menu items to become active

**Comment by shalvah** [comment]
Oh...I see. Let's revisit that later, perhaps. Better to not have that extra feature, than to have a broken core feature (sidebar).

**Pull request merged**